ecommerce/models.py

```diff
@@ -111,6 +111,61 @@ def type_string(self):
         """
         return self.content_type.model
 
+    @property
+    def title(self):
+        """
+        Helper property to return a string representation of the product title,
+        e.g.: "courserun.title", "program.title"
+
+        Returns:
+            str: String representing the product title
+        """
+        from courses.models import Program, CourseRun
+
+        content_object = self.content_object
+        if isinstance(content_object, Program):
+            return content_object.title
+        elif isinstance(content_object, CourseRun):
+            return content_object.course.title
+        else:
+            raise ValueError(f"Unexpected content type for {self.content_type.model}")
+
+    @property
+    def thumbnail_url(self):
+        """
+        Helper property to return a thumbnail url of the product.
```

Comment:
nit: please remove `Helper property to`

Reply:
this is how this file was written, I just followed the convention. For reference see property `type_string`